dashboard/package.json

```diff
@@ -9,10 +9,12 @@
     "@types/qs": "^6.5.1",
     "@types/react-jsonschema-form": "^1.0.4",
     "@types/react-select": "^1.2.6",
+    "@types/ws": "^6.0.0",
     "axios": "^0.17.1",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "js-yaml": "^3.10.0",
+    "mock-socket": "^8.0.2",
```

Comment:
isn't this a `devDependency`?

Reply:
It was not working for me in dev mode maybe because tests load a different env. 

UPDATE: Just tested it and it works as dev dependency! Good catch @andresmgot! Thanks!